python/lsst/analysis/tools/atools/refCatMatchPlots.py

```diff
@@ -642,3 +644,108 @@ def setDefaults(self):
             "AA1_tot": "mas",
             "AA1_sigmaMad_tot": "mas",
         }
+
+
+class TargetRefCatDeltaColorMetrics(AnalysisTool):
+    """Calculate the AB1 and ABF1 metrics from the difference between the
+    target and reference catalog coordinates.
+    """
+
+    parameterizedBand = Field[bool](
+        doc="Does this AnalysisTool support band as a name parameter", default=False
+    )
+    AB2 = Field[float](
+        doc="Value by which no more than ABF1% of measurements shall deviate from the mean.",
+        default=20,
```

Comment:
I was a bit confused by this docstring since from reading the code, it looks as though `AB2` is used when calculating what percent bright stars have a ra/dec/tot astrometry difference greater than AB2 (with AB2 expressed in milliarcseconds). Have I interpreted the code correctly, or am I way off the mark? If the former, then I think this docstring needs re-writing.

Reply:
You have interpreted it correctly. The statement from the SRD is "No more than ABF1 % of the measurements will deviate by more than AB2 milliarcsec from the mean", so this was my attempt to invert the statement to describe AB2. 
How about "Threshold at which to calculate the percentage of measurements (the ABF1 metric) that are outliers from the mean."?

Reply:
How about: "Separation in milliarcseconds that is used when calculating the ABF1 metric. ABF1 is the fraction of sources whose astrometric separation-from-reference is greater than SB2 milliarcseconds from the mean."?

Reply:
I went with yours with some small changes.